app/main.py

```diff
@@ -1,2 +1,111 @@
+from typing import Any
+
+
+class HashNode:
+    def __init__(
+            self, key: int | float | complex | str | tuple | object,
+            value: Any
+    ) -> None:
+        self.key = key
+        self.value = value
+        self.hash = hash(self.key)
+
+
 class Dictionary:
-    pass
+
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.loaded = int(self.capacity * (2 / 3))
+        self.hash_table = [None] * self.capacity
+
+    def get_hash_table_load_info(self) -> str:
+        return (
+            "is Load" if self.loaded == self.hash_table.count(None)
+            else "is not Load"
```

Comment:
The logic in `get_hash_table_load_info` is incorrect. The condition `self.loaded == self.hash_table.count(None)` is not a proper check for load factor. It should compare the number of non-None entries to the load factor threshold.